app/models/comment.rb

```diff
@@ -9,4 +10,21 @@ class Comment < ActiveRecord::Base
   def self.with_associations
     includes(:user).with_votes
   end
+
+  def question
+    if comment_on_type == 'Answer'
+      comment_on.question
+    else
+      comment_on
+    end
+  end
```

Comment:
This method doesn't seem to be used anywhere. Is there any need for it.
Also modifying it's name (to, say, `:get_question`) might be desirable.